adding Resources docs in TMC (#137)
terramate-io/terramate-docs: cloud/resources/index.md

---
title: Manage Resources | Terramate Cloud

> description: Learn how Terramate helps you to manage infrastructure Resources in Terramate Cloud

Reviewer: ```suggestion
description: Learn how to use the resource browser in Terramate Cloud to keep an overview of your entire infrastructure footprint managed by multiple teams, environments, and repositories.
```

---

# Resources

Resources in the Terramate Cloud dashboard lets teams manage all their infrastructure assets from a single place. It monitors resources deployed with Infrastructure as Code (IaC) across multiple VCS providers, repositories, and teams, offering a complete picture of your infrastructure. The dashboard displays resource status, enabling teams to identify drifted or misconfigured assets based on deployment outcomes and drift runs.

![Resources in Terramate Cloud](../assets/Resources_TMC.png)

The Resources page displays all known resources across multiple repositories. Each resource highlights its essential metadata. You can sort the list—by default, resources are arranged by the time of their last update—and use filters to narrow results by repository or status. Additionally, you can search for resources by entering a search string in the name or path. Clicking on any resource opens its [Resource Details](./details.md) page.

## Filter Options:

- **Status**: Filter resources by their current state:
    - `Healthy`: The resource is deployed successfully, and its current state matches the expected configuration.

>     - `Drifted`: The resource’s actual state has deviated from its declared configuration due to changes or misconfigurations.

Reviewer 2: ```suggestion
    - `Drifted`: The resource drifted. Which means that actual state as deployed in the cloud does not match the desired state as declared in the configuration. This can happen through manual changes, conflicting configurations, provider issues, or similar events.
```

>     - `Pending`: The resource is awaiting evaluation, with its status still being determined through ongoing deployment or drift checks.

Reviewer 3: Pending status happens when the resource is being deployed from IaC first time, so until next deployment or next drift check where the updated infrastructure data will be retrieved, we cannot know whether the resource status is healthy or drifted. I hope that makes sense. I proposed a suggestion but feel free to make it clearer.
```suggestion
    - `Pending`: The resource is being deployed, with its status still being determined through the end of current deployment or next following drift checks.
```

- **Technology**: Filter resources based on the technology used, such as Terraform or OpenTofu.

> - **Type**: Filter resources by their specific type.

Reviewer 2: what is a type?

- **Repository**: Filter resources by the repository that contains the resource.
- **Provider**: Filter resources by the cloud provider associated with them.

> - **Target**: Filter resources by the target environment or module where they are deployed.

Reviewer 4: This still mentions the `module` part like in the details page before. I would also remove the ", clarifying its role." part, just 
```suggestion
- **Target**: Specifies the environment where the resource is deployed.
```
